app/src/main/java/com/github/se/eduverse/ui/others/setting/Setting.kt

```diff
@@ -0,0 +1,112 @@
+package com.github.se.eduverse.ui.others.setting
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SettingsScreen(onSaveClick: () -> Unit, onCancelClick: () -> Unit) {
+  var privacySettings by remember { mutableStateOf("") }
+  var selectedTheme by remember { mutableStateOf("Light") }
+  var selectedLanguage by remember { mutableStateOf("English") }
+  var isThemeDropdownExpanded by remember { mutableStateOf(false) }
+  var isLanguageDropdownExpanded by remember { mutableStateOf(false) }
+
+  Column(
+      modifier = Modifier.fillMaxSize().padding(16.dp),
+      horizontalAlignment = Alignment.CenterHorizontally,
+      verticalArrangement = Arrangement.SpaceBetween) {
+        Column(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalAlignment = Alignment.CenterHorizontally) {
+              OutlinedTextField(
+                  value = privacySettings,
+                  onValueChange = { privacySettings = it },
+                  label = { Text(text = "Privacy Settings") },
+                  modifier = Modifier.fillMaxWidth().testTag("privacySettingsInput"))
+              Spacer(modifier = Modifier.height(8.dp))
+
+              // Dropdown for Theme Selection
+              Box(
+                  modifier =
+                      Modifier.fillMaxWidth().testTag("themeSelectionBox").clickable {
+                        isThemeDropdownExpanded = !isThemeDropdownExpanded
```

Comment:
The dropdowns for theme and language selection are well-implemented with appropriate testTags for testing. However, the clickable modifier is applied to the Box, which can be tricky for accessibility. Consider adding a more explicit clickable element like a Button or IconButton to ensure accessibility support.